package.json

```diff
@@ -39,7 +40,8 @@
     "prettier": "prettier \"ts_src/**/*.ts\" \"test/**/*.ts\" --ignore-path ./.prettierignore",
     "prettierjs": "prettier \"src/**/*.js\" --ignore-path ./.prettierignore",
     "test": "npm run build && npm run format:ci && npm run lint && npm run nobuild:coverage",
-    "unit": "npm run build && npm run nobuild:unit"
+    "unit": "npm run build && npm run nobuild:unit",
+    "generate:prefixes": "node scripts/generate-tagged-hash-prefixes.js && yarn prettierjs && yarn lint:fix"
```

Comment:
(fyi, we don't have yarn as a dev dependency)

I was thinking more along the lines of just replacing the existing code area with an object definition with `as const` or something.

Then adding a test to the crypto test file that will iterate through all the keys, calculate the hash (in the test) and assert it's equal.

Also, if we're going to do magic numbers anyways, using numbers instead of hex is fine. (`Buffer.from([1, 2, 3, 4])` etc.)

So basically, just taking the output of this script (maybe without using hex) and just copy pasting over the existing code as a const, then adding a test.

Thanks.

Reply:
I was thinking that someone may want to add another tag in future and decided to provide as script to simplify the process. But we can go we hardcoded variables